interfaces/repo.go

```diff
@@ -263,18 +263,14 @@ func (r *Repository) AddPlug(plug *Plug) error {
 	if i == nil {
 		return fmt.Errorf("cannot add plug, interface %q is not known", plug.Interface)
 	}
-	// Reject plug that don't pass interface-specific sanitization
```

Comment:
Why are we removing this?

Reply:
This is now done in info.ReadInfo (per our plan from the forum).